webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/MethodConditions.elm

```diff
@@ -212,3 +213,153 @@ parseOs os =
                           Nothing -> Nothing
                           x       -> Just (AIX { version = x })
     _ -> Nothing
+
+
+noVersion = {major = Nothing, minor = Nothing }
+
+osList : List (Maybe OS)
```

Comment:
I think that that list of OS etc should be provided from the server (even if it's `json` in a var in the page). 
It feels like we want to be able to update that list ni parallele with the list of supported OS from inventory (which should also be taken from a config file, but that's an other story). Else we risk to forket to update that part when updating supported os on the other side.